provider/ecs/ecs.go

```diff
@@ -319,6 +321,15 @@ func (e *ECS) CreateTaskDefinition(d service.Deploy) (*string, error) {
 					"awslogs-stream-prefix": aws.String(container.ContainerName),
 				},
 			})
+		} else if util.GetEnv("DATADOG_LOGS_ENABLED", "no") == "yes" {
+			containerDefinition.SetLogConfiguration(&ecs.LogConfiguration{
+				LogDriver: aws.String("json-file"),
+				Options:   map[string]*string{},
+			})
+			if containerDefinition.DockerLabels["com.datadoghq.ad.logs"] == nil {
+				datadogLogConfig := fmt.Sprintf(`[{ "source": "%s", "service": "%s" }]`, "docker", e.ServiceName)
```

Comment:
Given this is simple json I think this is fine, but if it was more complicated you might consider using a struct and converting it to json, along the lines of

```
  jstr, _ := json.Marshal(struct {
    Source  string  `json:"source"`
    Service string  `json:"service"`
  }{
    "docker",
    e.ServiceName,
  })
```